src/test/java/org/broadinstitute/hellbender/utils/tsv/TableReaderUnitTest.java

```diff
@@ -659,17 +680,29 @@ protected void processCommentLine(final String comment, final long ln) {
         Assert.assertEquals(actual, expected);
     }
 
-    private File createTestInput(final String... lines) throws IOException {
-        final File testFile = createTempFile("test", ".tab");
-        final PrintWriter testWriter = new PrintWriter(new FileWriter(testFile));
+    private Path createTestInput(final String... lines) throws IOException {
+        final Path testFile = createTempPath("test", ".tab");
+        fillTestInput(testFile, lines);
+        return testFile;
+    }
+
+    private Path createTestInputInMemory(final java.nio.file.FileSystem ramfs, final String... lines) throws IOException {
+        // We rely on the fact that the filesystem's in-memory so we don't need to delete.
+        final Path testFile = ramfs.getPath("test.tab");
+        fillTestInput(testFile, lines);
+        return testFile;
+    }
+
+    private void fillTestInput(final Path path, final String... lines) throws IOException {
+        final PrintWriter testWriter = new PrintWriter(Files.newBufferedWriter(path));
```

Comment:
I know this was just extracted, but might as well use try-with-resources.

Reply:
Sure thing, done.